src/ign-frontend/src/components/Demo.vue

```diff
@@ -262,7 +262,7 @@ export default Vue.component('Demo', {
         });
         document.querySelector('.preview').classList.toggle('processing');
 
-        if (conversionCount !== 0 && conversionCount % 3 === 0) {
+        if (conversionCount === 0 || conversionCount % 3 === 0) {
```

Comment:
What was the problem here?

Reply:
The modal asking for donation was visible after the 4th conversion.
With this change it'll be visible the first time then every 3rd conversion.

Reply:
ok, make sense